cohort-measure/src/main/java/com/ibm/cohort/measure/helpers/MeasurementPeriodHelper.java

```diff
@@ -1,10 +1,10 @@
 /*
```

Comment:
I know this isn't your change, but the `helpers` package seems pretty empty considering there are other classes with the "Helper" suffix floating around. Think it's worth it to either delete the package, or try to group other helpers together?

Reply:
I moved this helper to the parent `measure` package and deleted the `helper` package.